code/ai/aigoals.cpp

```diff
@@ -1940,7 +1940,7 @@ ai_achievability ai_mission_goal_achievable( int objnum, ai_goal *aigp )
 				aigp->ai_submode = ship_find_subsys( &Ships[sindex], aigp->docker.name );
 				aigp->flags.remove(AI::Goal_Flags::Subsys_needs_fixup);
 			} else {
-				Int3();
+				UNREACHABLE("Target name %s is not an arrived ship!", aigp->target_name);
```

Comment:
I need to look at this when I've had more coffee, but I'm not convinced this should be an UNREACHABLE yet.

Reply:
I think this situation is ruled out by the rest of the logic.  It would be cleaner to refactor this entire function to use the ship registry, but that's a topic for another PR.